inst/extdata/energy/A24.globaltech_coef.csv

```diff
@@ -0,0 +1,8 @@
+# File: A24.globaltech_coef.csv,,,,,
```

Comment:
Are these commas removed manually from the original csv? That file is used in multiple chuncks.